pkg/archive/changes_unix_test.go

```diff
@@ -0,0 +1,14 @@
+//go:build !windows
+
```

Comment:
Strictly speaking,
1. it should be `unix` rather than `!windows` (otherwise it won't work on e.g. plan9).
2. the build tag is redundant as we have `_unix_test.go` in the file name (supported since Go 1.19)

Reply:
I'll have to remember that.  Thanks!